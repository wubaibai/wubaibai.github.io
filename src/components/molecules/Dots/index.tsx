import React from 'react';

import Dot from 'components/atoms/Dot';

import style from './index.css';

interface DotsProperty {
	count: number;
}

const Dots: React.FC<DotsProperty> = ({ count }) => {
	const dots = [];
	for (let i = 0; i < count; i += 1) {
		dots.push(<Dot key={i} />);
	}

	return <div className={style.dots}>{dots}</div>;
};

export default Dots;
