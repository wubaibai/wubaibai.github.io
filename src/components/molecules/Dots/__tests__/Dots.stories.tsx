import React from 'react';
import { storiesOf } from '@storybook/react';

import Dots from 'components/molecules/Dots';

const stories = storiesOf('molecules/Dots', module);

stories.add('__interactive', () => <Dots count={3} />, { jest: 'Dots' });
