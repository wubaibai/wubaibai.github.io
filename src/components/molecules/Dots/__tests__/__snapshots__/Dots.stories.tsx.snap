// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`__interactive 1`] = `
<DocumentFragment>
  <div
    class="dots"
  >
    <div
      class="dot"
    />
    <div
      class="dot"
    />
    <div
      class="dot"
    />
  </div>
</DocumentFragment>
`;
